README.rst

```diff
@@ -1,3 +1,3 @@
 PyModelarDB
 ===========
-`PyModelarDB <https://github.com/ModelarData/PyModelarDB>`_ is the official Python PEP-249 client for `ModelarDB <https://github.com/ModelarData/ModelarDB>`_ and `MiniModelarDB <https://github.com/ModelarData/MiniModelarDB>`_.
+`PyModelarDB <https://github.com/ModelarData/PyModelarDB>`_ is the official Python PEP-249 client for ModelarDB. It supports both the current Rust-based version of `ModelarDB <https://github.com/ModelarData/ModelarDB-RS>`_ and the legacy JVM-based version of `ModelarDB <https://github.com/ModelarData/ModelarDB>`_.
```

Comment:
Now there are two different links which both have the same text, ModelarDB. Would it be better to make all of 'the current Rust-based version of ModelarDB' and all of 'the legacy JVM-based version of ModelarDB' part of the links?